src/components/pages/user/edit/tabs/app-user/web-user-form.tsx

```diff
@@ -0,0 +1,63 @@
+import { ArrowBackIcon } from "@chakra-ui/icons";
+import { Button } from "@chakra-ui/react";
+import { TextBoxField } from "@components/form/text";
+import { yupResolver } from "@hookform/resolvers/yup";
+import { generatePassword } from "@utils/basic";
+import useTranslation from "next-translate/useTranslation";
+import React, { useEffect, useState } from "react";
+import { FormProvider, useForm } from "react-hook-form";
+import * as Yup from "yup";
+
+export default function WebUser({ user, setIsCreateWebUser, setPassword }) {
+  const { t } = useTranslation();
+
+  const [password] = useState(generatePassword(10));
```

Comment:
1) Define this constant value in constants.ts file as `DEFAULT_PASSWORD_LENGTH` and use it here.